src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/Example4.java

```diff
@@ -15,16 +15,18 @@
   </module>
 </module>
 */
+//non-compiled with javac: Compilable with Java11
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
 
 // xdoc section -- start
 import com.neurologic.http.impl.ApacheHttpClient; // OK
 import static java.awt.Button.A;
 import javax.swing.JComponent; // violation, wrong order, caused by above static import
-                               // all static imports comes at bottom
+// all static imports comes at bottom
```

Comment:
restore indentation, as it is linewrap from previous line.
or do  change comment to start from line start ` // violation above,`

Reply:
> restore indentation, as it is linewrap from previous line. or do change comment to start from line start ` // violation above,`

Done. I used `// violation above` as restoring the indentation was causing a checkstyle failure.